ros_buildfarm/config/doc_build_file.py

```diff
@@ -17,7 +17,9 @@
 DOC_TYPE_ROSDOC = 'rosdoc_lite'
 DOC_TYPE_MANIFEST = 'released_manifest'
 DOC_TYPE_MAKE = 'make_target'
-DOC_TYPES = [DOC_TYPE_ROSDOC, DOC_TYPE_MANIFEST, DOC_TYPE_MAKE]
+DOC_TYPE_EXTERNAL = 'external_site'
```

Comment:
I'm not sure if "external site" is best name for this doc type. The other types all describe the build mechanism not the built artifact. I would have expected something more like "docker_container" to similarly describe the how not the what.

Since we'll need to document the new documentation job type on https://github.com/ros-infrastructure/ros_buildfarm/blob/a9f0554840dda39272d76ff19f67c1675d77f8d6/doc/jobs/doc_jobs.rst#different-types-of-documentation that will probably be a good way to settle on a type name. It should match the description in documentation.

Reply:
I don't have a strong preference, but the current job is somewhat coupled to the deploy mechanism as a git commit happens (and it is expected to happen) within the container.